go/vt/vtctl/vtctl.go

```diff
@@ -2257,7 +2257,7 @@ func commandMoveTables(ctx context.Context, wr *wrangler.Wrangler, subFlags *fla
 
 	workflow := subFlags.String("workflow", "", "Workflow name. Can be any descriptive string. Will be used to later migrate traffic via SwitchReads/SwitchWrites.")
 	cells := subFlags.String("cells", "", "Cell(s) or CellAlias(es) (comma-separated) to replicate from.")
-	tabletTypes := subFlags.String("tablet_types", "", "Source tablet types to replicate from (e.g. PRIMARY, REPLICA, RDONLY). Defaults to -vreplication_tablet_type parameter value for the tablet, which has the default value of PRIMARY,REPLICA.")
```

Comment:
We also have a similar helptext in the v2 umbrella command `commandVRWorkflow` ...

Reply:
Good catch! I think I covered that here: https://github.com/vitessio/vitess/pull/10040/commits/52a76e89edc846660ab0a4f7ca5666f7a7248919

Reply:
That required a follow-up: https://github.com/vitessio/vitess/pull/10040/commits/1ecf139817991b78c994b892ec862b0111529b50